app/components/CommunityAbout/test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import CommunityAbout from '.';
+import { shallow } from 'enzyme';
+
+function setup({ communitySummary }) {
+  const props = {
+    communitySummary,
+  };
+  const output = shallow(<CommunityAbout {...props} />);
+  return { props, output };
+}
+
+describe('CommunityAbout component', () => {
+  it('does not render the HTML communitySummary if there is none', () => {
+    const { output } = setup({ communitySummary: undefined });
+    const element = output.find('.CommunityAbout__text').nodes[0];
```

Comment:
I think we don't need to do this node[0] stuff, there's a better API for it in enzyme.